openapi/v2/core-data.yaml

```diff
@@ -58,12 +49,9 @@ components:
         origin:
           description: "A Unix timestamp indicating when the reading was originated at the source device (can support nanoseconds)"
           type: integer
-        pushed:
-          description: "A Unix timestamp indicating when the reading was permanently pushed to a northside app/service"
-          type: integer
       required:
         - deviceName
-        - labels
+        - name
```

Comment:
I think `origin` should also be required.

Reply:
FYI, I have pinged @tonyespy to get his take on making this required. @cloudxxx8 your thoughts?

Reply:
Updated `origin` as required per discussion in PR https://github.com/edgexfoundry/go-mod-core-contracts/pull/264.

Reply:
IMHO, `Reading` doesn't require `origin`.  If its `origin` is empty, it means the value is the same as the `Event`'s.

Reply:
I DS WG meeting, we agreed Origin on Readings is required.

Reply:
Update `origin` as required in `BaseReading`.